web/src/app/stats/components/filter-bar/filter-bar.component.scss

```
@@ -18,8 +22,69 @@
+    min-width: 0;
+    padding: 0 8px;
+  }
+
+  .step-btn--today .step-label {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.10.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)